nx-dev/nx-dev/pages/community.tsx

```diff
@@ -11,158 +11,65 @@ import {
 } from '@nrwl/nx-dev/ui-community';
 import { NextSeo } from 'next-seo';
 import { useRouter } from 'next/router';
+import { packagesApi } from '../lib/api';
 
 declare const fetch: any;
 
+interface PluginInfo {
+  description: string;
+  name: string;
+  url: string;
+  isOfficial: boolean;
+}
 interface CommunityProps {
-  pluginList: {
-    description: string;
-    name: string;
-    url: string;
-    isOfficial: boolean;
-  }[];
+  pluginList: PluginInfo[];
 }
 
 export async function getStaticProps(): Promise<{ props: CommunityProps }> {
   const res = await fetch(
     'https://raw.githubusercontent.com/nrwl/nx/master/community/approved-plugins.json'
   );
   const pluginList = await res.json();
+
+  const officialPluginList = (
+    packagesApi.getPackageDocuments().itemList ?? []
+  ).filter(
+    (m) =>
+      m.id !== 'add-nx-to-monorepo' &&
+      m.id !== 'cra-to-nx' &&
+      m.id !== 'create-nx-plugin' &&
+      m.id !== 'create-nx-workspace' &&
+      m.id !== 'make-angular-cli-faster' &&
+      m.id !== 'tao'
+  );
+
   return {
     props: {
-      pluginList: pluginList.map((plugin) => ({
-        ...plugin,
-        isOfficial: false,
-      })),
+      pluginList: [
+        ...officialPluginList.map((plugin) => ({
+          name: `@nrwl/${plugin.name}`,
+          description: plugin.description ?? '',
+          url: plugin.path,
+          isOfficial: true,
+        })),
+        ...pluginList.map((plugin) => ({
+          ...plugin,
+          isOfficial: false,
+        })),
+      ].map((plugin: PluginInfo) => {
+        const shortenedDescription = plugin.description.substring(0, 180);
```

Comment:
Can you remove this? I would prefer doing this in CSS with a `truncate` on the card element itself. That way we keep all the keywords and the full description for accessibility and SEO.

Reply:
The line to update with the `truncate` would be this one: [nx-dev/ui-common/src/lib/plugin-card.tsx:37](https://github.com/nrwl/nx/blob/master/nx-dev/ui-common/src/lib/plugin-card.tsx#L37)

Reply:
Yeah let's use the plugin 👍